src/odemis/gui/plugin/__init__.py

```diff
@@ -494,6 +495,29 @@ def showProgress(self, future):
         # press the "cancel" button, if such button exists, otherwise provide
         # such a button. That button will call cancel() on the future.
 
+    @call_in_wx_main
+    def add_acquisition_info(self, txt=None, lvl=logging.INFO):
```

Comment:
Change the name to `set_acquisition_info`. (Because it doesn't always add, but changes the previous one, or even remove the previous text).